plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/deployimage/DeployImageWizard.java

```diff
@@ -61,14 +61,26 @@ public DeployImageWizard(IDockerImage image, Connection connection, IProject pro
 			IDockerConnection dockerConnection = image.getConnection();
 			model.setOriginatedFromDockerExplorer(true);
 			model.setDockerConnection(dockerConnection);
-			model.setImageName(image.repo());
+			model.setImageName(getImageNameWithTag(image));
 		}
 		model.setStartedWithActiveConnection(isAuthorized);
 		model.initModel(connection, project, isAuthorized);
 
 		setNeedsProgressMonitor(true);
 	}
 
+	private String getImageNameWithTag(IDockerImage image) {
```

Comment:
How does this get the tag we selected?  Doesn't this just get the first sorted one, which may or may not be the one i desire? Based on the issue, it looks like we should be getting a specific tag.